### DISCUSSION TITLE
Dark Mode Missing on Project

### DISCUSSION THREAD

**Discussion by aurexav** [community/community#18125] [Projects and Issues]
If I use *Dark Tritanopia* theme. The dark mode does not work on the project page.
If I use the default dark theme. Then it works well.

**Comment by MadhanDevlpr** [comment]
Can you please send us images on it

**Reply by aurexav** [reply]
> Can you please send us images on it

<img width="1491" alt="image" src="https://user-images.githubusercontent.com/16152305/173196893-71985973-2146-4f89-a9af-bdc89926fce6.png">
<img width="1475" alt="image" src="https://user-images.githubusercontent.com/16152305/173196949-563fdc77-ee3b-4753-b4f9-2bdd231db1f6.png">

**Comment by akbar-ardiansyah** [comment]
I think this might help a little
`//dark mode //check localstorage
if (localStorage.getItem('preferredTheme') == 'dark')
setDarkMode(true)

function setDarkMode(isDark) {
if (isDark) {
    document.body.setAttribute("id", "darkmode");
    localStorage.setItem('preferredTheme', 'dark');
} else {
    document.body.setAttribute("id", "");
    localStorage.removeItem('preferredTheme');
}
}
`
